loops-and-arrays.js

```diff
@@ -53,3 +67,17 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 24 is 13824',
 //   'Het volume van 11 is 1331'
 // ]
+
+
+console.log(squares);
+
+let volume = [];
+
+for (let i = 0; i < squares.length; i++) {
+    volume.push(squares[i]**3);
```

Comment:
This is very great work! The double ** is genius. Compliments! Consistent use of  ;  and everything is very readable.